### ISSUE TITLE
Only first track from album can be streamed to chromecast

### ISSUE THREAD

**Issue by agilob** [jellyfin/jellyfin#3530]
**Describe the bug**
Start steaming album to chromecast. Click/tap on any non-first track. Chromecast always starts playing first track.

**System (please complete the following information):**
 - Virtualization:  Docker, KVM, LXC]
 - Clients: Browser, Android
 - Browser: Chrome 80
 - Jellyfin Version:  10.5.5
 - Playback: dont know 
 - Installed Plugins: none
 - Reverse Proxy: traefik
 - Base URL: none
 - Networking: Host

**Comment by BaronGreenback** [comment]
Is this the PlayTo functionality? I ask as i've been working on that. PR #3536

**Comment by agilob** [comment]
It's on chromecast playing from google chrome or android app, not DLNA.

**Comment by mueslimak3r** [comment]
(this is a copy/paste of a post I made in the Jellyfin troubleshooting chat)
I am currently experiencing a similar issue with Jellyfin casting to Chromecast. I have tested this issue both using the web client and the android app. The issue is this: when attempting to play music from an album to the chromecast, the first song in the album is always played regardless of which song in the album is selected. On the android app the skip button often stops working too, and sometimes repeated attempts to play a song that isn't the first will result in the 'remote control' playback ui freezing up. I have also verified that when I am not connected to the chromecast everything works fine and I can play all the music in whichever order I like.
I have verified that both the android app and the server (debian on raspberry pi 4 4GB) are up to date
Also I'm using a reverse proxy via Caddy 2
